docs/diagrams/FileManagerDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+!include Styles.puml
+'https://plantuml.com/class-diagram
+
+Package "filemanager"  <<Rectangle>>{
+     Class Storage COLOR_STORAGE
+     Class InstrumentDecoder COLOR_STORAGE
+     Class InstrumentEncoder COLOR_STORAGE
+     Class CryptoDecoder COLOR_STORAGE
+     Class StockDecoder COLOR_STORAGE
+     Class EtfDecoder COLOR_STORAGE
+     Class ForexDecoder COLOR_STORAGE
+}
+
+Class outside COLOR_WHITE
+outside -down[COLOR_STORAGE]-> Storage
+CryptoDecoder -up[COLOR_STORAGE]->InstrumentDecoder
```

Comment:
to show inheritance need to use the open arrow.   ".|>" likewise for the model diagram